src/main/java/com/twitter/whiskey/net/Response.java

```diff
@@ -19,7 +19,7 @@
     private ByteBuffer body;
     private RequestStats stats;
 
-    Response(int statusCode, Headers headers, ByteBuffer body, RequestStats stats) {
+    public Response(int statusCode, Headers headers, ByteBuffer body, RequestStats stats) {
```

Comment:
I changed this to public since I required it for creating dummy responses. Ideally we would need a ResponseBuilder